README.md

```diff
@@ -57,6 +57,16 @@ The Summary element is comprised of:
  * The total number of alerts by type - i.e. Breach, Unknown, etc.
  * The number of new alerts by type (since the day before).
 
+You cannot check the progress of a portfolio file upload because these files load too quickly. Instead, when you upload a portfolio file, it will return a status immediately of any of the following:
+
+ValidationState  | Explanation  
+-----------------|-------------
+Uploaded         | Succesfully uploaded    
+Validation       | Validation failure
```

Comment:
Would it be clearer if it was `Validation Failure | Document did not pass validation` ? more question than suggestion